packages/patternfly-4/content/design-guidelines/usage-and-behavior/charts/design.md

```diff
@@ -57,25 +71,65 @@ Stacked bar charts describe totals while allowing a degree of internal breakdown
+
+If you need to compare one category to another, consider using a [Bar chart](#bar-chart).
 
 #### Usage
-The most common use cases for donut charts are to show progress completion as a percentage, show utilization for an object as a percentage and show the relationship of a set of values to a whole. Use donut charts to compare an individual section to the whole. As said previously, the entire donut should add up to 100% and the fill represents a percentage of that. There should be no more than six categories in total, similar to pie charts We recommend that you use the [Colors for charts](/design-guidelines/styles/chart-colors) to represent your data when thresholds are not present. Consider using a [Bar chart](#bar-chart) if you would like to compare one category to another.
+In donut charts, you can choose to use percentages or integer values to compare parts to the whole. When deciding which to use, consider the information that is most important to your user and what makes the most sense for your use case. For example, if a user knows they have 123 farm animals and they’re interested in knowing how many of those animals are cows, it probably makes more sense to use an integer value. If that same user is interested in knowing how much storage space they have left in their grain silo, a percentage might be better.
+
+
+For donut charts, don’t try to represent more than six categories. We recommend using the [Colors for charts](/design-guidelines/styles/chart-colors) guidelines to represent your data when thresholds are not present.
 
 #### Example
-![Donut chart](./img/donutchart.png)
+![Donut chart](./img/donut-chart.png)
+
+1. **Segment fill**: We recommend using [Colors for charts] (/design-guidelines/styles/chart-colors) for different items within the donut chart.
+2. **Segment padding**:  Always provide 3px of padding between segments.
+3. **Chart tooltip**: A tooltip will appear upon hover that states the name of the segment and corresponding value. For example, if the segment represents “Bugs,” and the value being represented is 25, your chart tooltip would state, “Bugs: 25.”
+4. **Label**: When the Donut Chart is contained within a dashboard card, there is a label defining what the Donut Chart represents. The label may also represent the total value of the data set. If this optional representation is chosen, it should follow the format of [total numeric value] + [data set label]. The total numeric value cannot contain more than 14 characters. The data set label cannot contain more than 24 characters. They are to be centered within the donut and styled as shown. The label should be using 24px font in standard text color with 14px font underneath in secondary text color. If the label does not fit within the donut, place it outside of the donut and leave the center empty.
+5. **Legend**: In order to be compliant with accessibility guidelines, a legend is necessary.
```

Comment:
I was wondering about the numbers to the right of the legend categories, e.g. "bugs:25". Is the intent to report the current value as part of the legend? I guess that's OK, but it surprised me a bit because we don't do that for other charts. Maybe need to explain better.

Reply:
Yeah for accessibility reasons, the intention is to do that for all legends. Obviously that info is customizable though

Reply:
Makes sense. Should all of the legends be updated this way then, I asked because I was only seeing this for donut charts.